aliyun/src/test/java/org/apache/iceberg/aliyun/oss/mock/TestLocalAliyunOSS.java

```diff
@@ -69,6 +70,9 @@ public void after() {
 
   @Test
   public void testBuckets() {
+    Assume.assumeTrue("Aliyun integration test cannot delete existing bucket from test environment.",
+        OSS_TEST_RULE.getClass() == AliyunOSSMockRule.class);
```

Comment:
Add this sentence because we don't want to run this test cases for integration test, because it will delete the bucket from test environment, while the test environment may have other objects that were used by other tests.

Reply:
In AWS we just assumes the bucket exists, and create a random prefix as the root for all tests. By doing that we don't deal with bucket deletion, which is quite dangerous as you say. Would that be better also for Aliyun?

Reply:
> In AWS we just assumes the bucket exists, and create a random prefix as the root for all tests.

The aliyun integration tests is using the same approach as you said.  This suite case was designed to guarantee that the Aliyun Mock OSS Application will have the same semantics as the production aliyun OSS services.  If both AliyunOSSMockRule and OSSIntegrationTestRule are successfully applied to this case, then we can ensure that all the unit tests which are built on top of the AliyunOSSMockRule are correctly designed.

As we iceberg FileIO won't depend any bucket creation or deletion interfaces , plus the dangerous bucket deletion behavior.  So Here I disabled the buckets tests for integration tests.